python/orca/test/bigdl/orca/learn/spark/test_estimator_for_bigdl.py

```diff
@@ -27,7 +27,7 @@
 from bigdl.orca.learn.bigdl import Estimator
 from bigdl.dllib.optim.optimizer import Adam, SGD, ValidationSummary
 from bigdl.dllib.keras import layers as ZLayer
-from bigdl.dllib.keras.models import Model as ZModel
+from bigdl.dllib.keras.model import Model as ZModel
```

Comment:
This is not correct; it should be either `from bigdl.dllib.keras import Model` or `from bigdl.dllib.keras.models import Model`

Reply:
This pr is fix the exceptions in nightly build jekins. The pr for package rename to `from bigdl.dllib.keras import Model` is on progress.